R/module_nested_tabs.R

```diff
@@ -250,7 +250,10 @@ srv_nested_tabs.teal_module <- function(id, datasets, modules, is_module_specifi
   datanames <- if (is.null(module$datanames) || identical(module$datanames, "all")) {
     datasets$datanames()
   } else {
-    unique(module$datanames) # todo: include parents! unique shouldn't be needed here!
+    include_parent_datanames(
+      unique(module$datanames),
```

Comment:
Move the `uniqe` to `include_parent_datanames`?
```
 for (i in unique(dataname)) {
  ...
```

Reply:
I noticed unique isn't even needed as `include_parent_datanames` returns unique datanames